src/pages/AdminLoginPage/AdminLoginPage.css

```diff
@@ -0,0 +1,24 @@
+.password-wrapper .toggle-password {
+  position: absolute;
+  top: 50%;
+  right: 1rem;
+  transform: translateY(-50%);
+  cursor: pointer;
+}
+x .password-input-container {
+  position: relative;
+}
+
+.CopyIcon {
```

Comment:
What does this do? You notice you copied code without understanding what it even does?